RevenueCatUI/CustomerCenter/Data/CustomerCenterData.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  File.swift
+//  
+//
+//  Created by Cesar de la Vega on 28/5/24.
+//
+
+import Foundation
+
+struct CustomerCenterData: Decodable {
+
+    let id: String
+    let paths: [HelpPath]
+    let title: LocalizedString
+    let supportEmail: String
+    let appearance: Appearance
+
+    enum HelpPathType: String, Decodable {
+        case missingPurchase = "MISSING_PURCHASE"
+        case refundRequest = "REFUND_REQUEST"
+        case changePlans = "CHANGE_PLANS"
+        case cancel = "CANCEL"
+        case unknown
+    }
+
+    struct HelpPath: Decodable {
+
+        let id: String
+        let title: LocalizedString
+        let type: HelpPathType
+        let promotionalOffer: PromotionalOffer?
+        let feedbackSurvey: FeedbackSurvey?
+
+    }
+
+    struct LocalizedString: Decodable {
+
+        let enUS: String
+
+    }
+
+    struct PromotionalOffer: Decodable {
+
+        let iosOfferId: String
+        let eligibility: Eligibility
+
+    }
+
+    struct Eligibility: Decodable {
+
+        let firstSeen: String
+
+    }
+
+    struct FeedbackSurvey: Decodable {
+
+        let title: LocalizedString
+        let options: [FeedbackSurveyOption]
+
+    }
+
+    struct FeedbackSurveyOption: Decodable {
+
+        let id: String
+        let title: LocalizedString
+        let promotionalOffer: PromotionalOffer?
+
+    }
+
+    struct Appearance: Decodable {
+
+        let mode: String
+        let light: String
+        let dark: String
```

Comment:
This should use something similar to `PaywallColor`

Reply:
I will make these changes in another PR so I keep this lean and easy to review. Next PR will take of calling the API and deserialization

Reply:
Maybe we could
`typealias RCColor = PaywallColor`
or change the name of the actual class and reverse the typealias for backward compatibility.